clients/fides-js/src/components/fides.css

```diff
@@ -559,18 +577,28 @@ div#fides-banner-inner .fides-privacy-policy {
   width: 100%;
   display: flex;
   justify-content: space-between;
-  margin-right: 0.5em;
   min-height: 40px;
   align-items: center;
+
+  & svg {
```

Comment:
I _think_ this kind of nesting doesn't work in firefox 😢 

see firefox on left, chrome on right
![image](https://github.com/ethyca/fides/assets/24641006/db8e297c-f881-4be6-a7de-dda234c7978f)

Reply:
I'm on firefox 116 (which is a little out of date) which doesn't support it, seems to only be a recent thing https://caniuse.com/css-nesting